Packages/com.unity.inputsystem/InputSystem/Editor/UITKAssetEditor/Views/CopyPasteHelper.cs

```diff
@@ -1,6 +1,7 @@
 #if UNITY_EDITOR && UNITY_INPUT_SYSTEM_PROJECT_WIDE_ACTIONS
 using System;
 using System.Collections.Generic;
+using System.Linq;
```

Comment:
The new guidelines recommend to not use Linq anymore. I know it is on various places in our code already, but I think it makes sense to try not using it in the new/changed code.

Reply:
The "Select()" call from `bindingsForAction.Select`, which I copied over from `/UITKAssetEditor/Views/Selectors.cs` is what's using it.

I'll restore the function `Selectors.GetLastBindingIndexForSelectedAction()`, which contained this logic (Selectors.cs already uses LINQ) and call into that from here.

Reply:
Moved the code using LINQ back into Selectors.cs with: 630fe8d551ecfe45a0245c77583f76b8faf1f285